src/domain/entities/Race/Lefeu/Lefeu.ts

```diff
@@ -0,0 +1,44 @@
+import {SelectableAttributesRace} from '../../SelectableAttributesRace';
+import type {Attribute, Attributes} from '../../Sheet/Attributes';
+import {type SkillName} from '../../Skill';
+import {RaceName} from '../RaceName';
+import {Deformity} from './Deformity/Deformity';
+import {SonOfTormenta} from './SonOfTormenta';
+
+export class Lefeu extends SelectableAttributesRace {
+	static raceName = RaceName.lefeu;
+	static attributeModifiers: Partial<Attributes> = {charisma: -1};
+
+	readonly abilities = {
+		sonOfTormenta: new SonOfTormenta(),
+		deformity: new Deformity(),
+	};
+
+	/**
+ * Returns an instance of lefeu race.
+ * @param selectedAttributes - 2 different attributes
+ * @param deformity - +2 on 2 skills
+  **/
+
+	constructor(selectedAttributes: Attribute[]) {
+		super(selectedAttributes, RaceName.human);
```

Comment:
E se a gente já deixasse o lefeu escolher uma raça?

```suggestion
        private previousRace: RaceName 
        
	constructor(selectedAttributes: Attribute[], previousRace: RaceName = RaceName.human) {
		super(selectedAttributes, RaceName.human);
```